### PR TITLE
update readme to v3.0.0.

### PR DESCRIPTION
**Pull request by FeynmanZhou**
Signed-off-by: FeynmanZhou <pengfeizhou@yunify.com>

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>

/kind documentation

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for reviewers**:
```
```

**Additional documentation, usage docs, etc.**:

```docs

```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,112 +51,65 @@ KubeSphere uses a loosely-coupled architecture that separates the [frontend](htt
...
+[Quickstarts](https://kubesphere.io/docs/quick-start/) include six hands-on lab exercises that help you quickly get started with KubeSphere.
 
-- `Kubernetes version`： `1.15.x, 1.16.x, 1.17.x`
-- `2.10.0 ≤ Helm Version ＜ 3.0.0` excluding 2.16.0 because of [#6894](https://github.com/helm/helm/issues/6894). Please see [Install and Configure Helm in Kubernetes](https://devopscube.com/install-configure-helm-kubernetes/). Helm v3 will be supported in KubeSphere 3.0.0.
-- An existing Storage Class in your Kubernetes cluster, use `kubectl get sc` to verify it
-- The CSR signing feature is activated in kube-apiserver, see [RKE installation issue](https://github.com/kubesphere/kubesphere/issues/1925#issuecomment-591698309).
+### Installing on Existing Kubernetes Cluster
 
-Install KubeSphere using kubectl.
+- [Installing KubeSphere on Amazone EKS](https://kubesphere.io/docs/installing-on-kubernetes/hosted-kubernetes/install-kubesphere-on-eks/)
```
Comment (rayzhou2017) [inline README.md:84]: its better to point to the  specific version link since this URL will be changed to newer version.

Comment (FeynmanZhou) [reply README.md:84]: How about replacing the URL to `https://v3-0.docs.kubesphere.io/docs/`?

Comment (rayzhou2017) [reply README.md:84]: yes that's what I am talking about.

Comment (FeynmanZhou) [reply README.md]: Done

File: README.md [hunk 2]
```diff
@@ -51,112 +51,65 @@ KubeSphere uses a loosely-coupled architecture that separates the [frontend](htt
...
 | Kubernetes Resource Management | Provide web console for creating and managing Kubernetes resources, with powerful observability including monitoring, logging, events, alerting and notification |
 | DevOps System | Provide out-of-box CI/CD based on Jenkins, and offers automated workflow tools including binary-to-image (B2I) and source-to-image (S2I) |
 | Application Store | Provide application store for Helm-based applications, and offers application lifecycle management |
 | Service Mesh (Istio-based) | Provide fine-grained traffic management, observability and tracing for distributed microservice applications, provides visualization for traffic topology |
-| Rich Observability | Provide multi-dimensional monitoring metrics, and provides multi-tenant log query and collection, support alerting and notification for both application and infrastructure |
+| Rich Observability | Provide multi-dimensional monitoring metrics, and provides multi-tenant logging, events and [auditing](https://kubernetes.io/docs/tasks/debug-application-cluster/audit/) management, support alerting and notification for both application and infrastructure |
```
Comment (rayzhou2017) [inline README.md:59]: It is better to check all links and change them to v3 doc if possible.

**Pull request merged**